dao/src/main/kotlin/repositories/ortrun/DaoOrtRunRepository.kt

```diff
@@ -109,11 +110,21 @@ class DaoOrtRunRepository(private val db: Database) : OrtRunRepository {
             OrtRunDao.listQuery(parameters, OrtRunDao::mapToModel) {
                 var condition: Op<Boolean> = Op.TRUE
 
-                filters?.status?.let { statusFilter ->
-                    condition = condition and OrtRunsTable.status.applyFilter(
-                        statusFilter.operator,
-                        statusFilter.value
-                    )
+                filters?.let {
```

Comment:
I don't strictly object against this approach with generic filter conditions, but I am worried that we add here significant complexity just to duplicate logic/functionality the ORM framework already contains. Wouldn't it be more straight-forward to add a dedicated `list` function to `OrtRunRepository` which applies the filtering logic for old ORT runs?

Reply:
You right, no need to expose such filtering in the API.